CONTRIBUTING.md

```diff
@@ -3,3 +3,9 @@
 To contribute to Zinc, you or your employer must sign Lightbend Contributor License Agreement <https://www.lightbend.com/contribute/cla>.
 
 If you add a new file, run `createHeaders` task from sbt shell to put the copyright notice.
+
+### To build with other module source
```

Comment:
This emulates monorepo. When we change util (or librarymanagement) and zinc together, this lets us compile them together. (Might still require publishLocal for testing)